examples/tcp_client.rs

```diff
@@ -1,36 +1,155 @@
-use std::{io, net};
```

Comment:
I would appreciate not changing this example and instead just wrapping the socket error bits in the configuration guards.

Reply:
Don't modify the Example too much; just make it so the `is_err`-specific code is guarded.